ThirdPartyNotices.txt

```diff
@@ -12,9 +12,10 @@ bring it to our attention. Post an issue or email us:
 The attached notices are provided for information only.
 
 1. bazelbuild/bazel (https://github.com/bazelbuild/bazel/)
-2. force-net/crc32.net (https://github.com/xamarin/proguard/)
-3. google/desugar (https://android.googlesource.com/platform/external/desugar/+/master/)
-4. nunit/nunitlite (https://github.com/nunit/nunitlite/)
+2. force-net/crc32.net (https://github.com/force-net/Crc32.NET)
+3. google/aosp (https://source.android.com/setup/start/licenses)
```

Comment:
Instead of `google/aosp`, this should be `android/platform/tools/base`

Reply:
Additionally, the URL shouldn't be the URL to the license -- `/setup/start/licenses` -- but instead to *where the source code was obtained*, in this case (presumably) https://android.googlesource.com/platform/tools/base/+/d41d662dbf89f9b60ca6256415a059c0107749b8/sdk-common/src/main/java/com/android/ide/common/packaging/PackagingUtils.java or some "higher-level" URL like https://android.googlesource.com/platform/tools/base.